presto-main/src/test/java/io/prestosql/execution/TestCallTask.java

```diff
@@ -107,6 +115,27 @@ public void testExecuteNoPermission()
         assertThat(invoked).isFalse();
     }
 
+    @Test
+    public void testExecuteNoPermissionOnInsert()
+    {
+        TransactionManager transactionManager = createTransactionManager();
```

Comment:
There is plenty of code copied from `testExecuteNoPermission` can you please somehow unify that?